build.gradle.kts

```diff
@@ -228,7 +228,11 @@ tasks {
       println("Cached $buildCodyDir")
       return buildCodyDir
     }
-    val codyDir = unzipCody()
+    val codyDirEnv: String? =
+        System.getenv("CODY_DIR")?.replace("~", System.getProperty("user.home"))
```

Comment:
for some reason `~` does not work 🙅

Reply:
You could consider reading it from properties file as well.
It would give you more flexibility with overriding:

1) using env var:
ORG_GRADLE_PROJECT_codyDir=xxx

2) inline when calling `gradle`command:
gradle -PcodyDir=xxx

3) override in `gragle.properies` is you want to make it pernament in a given workspace